packages/rancher-monitoring/generated-changes/overlay/charts/grafana/templates/rancher-monitoring/service-clusterrole.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.global.rbac.create .Values.global.rbac.userRoles.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: monitoring-grafana-view
+  namespace: {{ template "kube-prometheus-stack.namespace" . }}
+  labels: {{ include "kube-prometheus-stack.labels" . | nindent 4 }}
+  {{- if .Values.global.rbac.userRoles.aggregateToDefaultRoles }}
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  {{- end }}
```

Comment:
From design meeting: aggregation needs to be removed. We'll need to create a role template in Rancher to allow users to bind to this role instead.